src/content/docs/features/window-customization.mdx

```diff
@@ -2,13 +2,207 @@
+
+#### tauri.conf.json
+
+To remove the default titlebar, you'll need to set `decorations` to `false` in your `tauri.conf.json`:
+
+```json title="tauri.conf.json" {4}
+"tauri": {
+	"windows": [
+		{
+			"decorations": false
+		}
+	]
+}
+```
+
+#### Permissions
+
+To make calls to `appWindow` work, don't forget to add window permissions in capability file:
+
+```json title="src-tauri/capabilities/main.json" ins={7-8}
+{
+	"$schema": "../gen/schemas/desktop-schema.json",
+	"identifier": "main-capability",
+	"description": "Capability for the main window",
+	"windows": ["main"],
+	"permissions": [
+		"window:default",
+		"window:allow-start-dragging",
+	]
+}
+```
+
+| Permission                              | Description                                                                    |
+|-----------------------------------------|--------------------------------------------------------------------------------|
+| `window:default`                        | Default permissions for the plugin. Except `window:allow-start-dragging`.      |
+| `window:allow-close`                    | Enables the close command without any pre-configured scope.                    |
+| `window:allow-minimize`                 | Enables the minimize command without any pre-configured scope.                 |
+| `window:allow-start-dragging`           | Enables the start_dragging command without any pre-configured scope.           |
+| `window:allow-toggle-maximize`          | Enables the toggle_maximize command without any pre-configured scope.          |
+| `window:allow-internal-toggle-maximize` | Enables the internal_toggle_maximize command without any pre-configured scope. |
+
+#### CSS
+
+You'll need to add some CSS for the titlebar to keep it at the top of the screen and style the buttons:
+
+```css
+.titlebar {
+	height: 30px;
+	background: #329ea3;
+	user-select: none;
+	display: flex;
+	justify-content: flex-end;
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+}
+.titlebar-button {
+	display: inline-flex;
+	justify-content: center;
+	align-items: center;
+	width: 30px;
+	height: 30px;
+	user-select: none;
+	-webkit-user-select: none;
+}
+.titlebar-button:hover {
+	background: #5bbec3;
+}
+```
+
+#### HTML
+
+Now, you'll need to add the HTML for the titlebar. Put this at the top of your `<body>` tag:
+
+```html
+<div data-tauri-drag-region class="titlebar">
+	<div class="titlebar-button" id="titlebar-minimize">
+		<img
+			src="https://api.iconify.design/mdi:window-minimize.svg"
+			alt="minimize"
+		/>
+	</div>
+	<div class="titlebar-button" id="titlebar-maximize">
+		<img
+			src="https://api.iconify.design/mdi:window-maximize.svg"
+			alt="maximize"
+		/>
+	</div>
+	<div class="titlebar-button" id="titlebar-close">
+		<img src="https://api.iconify.design/mdi:close.svg" alt="close" />
+	</div>
+</div>
+```
+
+Note that you may need to move the rest of your content down so that the titlebar doesn't cover it.
```

Comment:
Maybe a note aside :::? Not sure, maybe if it looks better and not out of place

Reply:
Using aside looks a bit over to me. I would prefer to leave it plain.